pkg/config/schemas/cmd/schemagen/main.go

```diff
@@ -0,0 +1,254 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A simple program that consumes a YAML file describing Istio schemas and produces as output
+// a Go source file providing variable definitions for those schemas.
+
+package main
```

Comment:
Curious about why you picked a different approach here (schemas/cmd/schemagen v.s. schemas/schemagen.main.go).

I don't have a strong preference either way but was slightly preferring all the generation related code (& data) to be co-located.